test/testHelpers/collections/collectionHelper.ts

```diff
@@ -6,15 +6,15 @@ const COLLECTION_ID = 11111
 const COLLECTION_ALIAS_STR = 'secondCollection'
 const COLLECTION_NAME_STR = 'Laboratory Research'
 const COLLECTION_AFFILIATION_STR = 'Laboratory Research Corporation'
-const COLLECTION_DESCRIPTION_STR = 'This is an example collection used for testing.'
-
+const COLLECTION_DESCRIPTION_HTML = 'This is an <b>example</b> collection used for testing.'
+const COLLECTION_DESCRIPTION_MARKDOWN = 'This is an **example** collection used for testing.'
 export const createCollectionModel = (): Collection => {
   const collectionModel: Collection = {
     id: COLLECTION_ID,
     alias: COLLECTION_ALIAS_STR,
     name: COLLECTION_NAME_STR,
     affiliation: COLLECTION_AFFILIATION_STR,
-    description: COLLECTION_DESCRIPTION_STR,
+    description: COLLECTION_DESCRIPTION_HTML,
```

Comment:
Shouldn't this be the `COLLECTION_DESCRIPTION_MARKDOWN` since this is the transformed description?
```suggestion
    description: COLLECTION_DESCRIPTION_MARKDOWN,
```

Reply:
thanks, much cleaner to do it this way, we only need one model for the tests